### DISCUSSION TITLE
有关数据库

### DISCUSSION THREAD

**Discussion by ghost** [hydro-dev/Hydro#296] [Q&A] [answered]
SU 用户权限都设为 root 了还没发改别人发的评论，然后想用数据库改结果没发现讨论区的数据库，求助，在线等。

**Comment by Kanna-Neko** [comment]
建议你使用https://github.com/hydro-dev/Hydro/blob/master/packages/hydrooj/src/model/discussion.ts 中的脚本实现该功能

**Comment by ghost** [comment, marked as answer]
thx，找到了，没看到 MongoDB Compass 要翻页，怪我眼睛不好。在 document 表中的 docType:21 是讨论， docType:22 是回复
